javascript-arrays/main.js

```diff
@@ -0,0 +1,31 @@
+var colors = ['red', 'white', 'blue'];
+
+console.log('value of colors[0]', colors[0]);
+console.log('value of colors[1]', colors[1]);
+console.log('value of colors[2]', colors[2]);
+
+var America = colors[0] + ', ' + colors[1] + ', ' + 'and' + ' ' + colors[2];
+console.log('America is', America);
+
+colors[2] = 'green';
+
+var Mexico = colors[0] + ', ' + colors[1] + ', ' + 'and' + ' ' + colors[2];
+console.log('Mexico is', Mexico);
+
+console.log('colors', colors);
+
+var students = ['Leonardo', 'Donatello', 'Raphael', 'Michaelangelo'];
+
+var numberOfStudents;
+numberOfStudents = students.length;
```

Comment:
You don't need to do this in JavaScript, you can just assign values on the same line you declare a variable